### PR TITLE
 Edited Command_wildcard. ISSUE #1994

### PR DESCRIPTION
**Pull request by DarkGamingDronze**
Added some commands to the blacklist of /wildcard

### CODE REVIEW

**Comment by Wild1145** [comment]
Thank you for contributing to the project, could you please amend your opening message to include the issue that this PR resolves. Thank You.

**Comment by DarkGamingDronze** [comment]
I do not hnderstand you. Please explain what you mean - I am new and this is my first pull request ever since the new tfm came out.

**Comment by Wild1145** [comment]
@DarkGamingDronze Take a look at the Contributing Guidelines, that should explain everything you need to know.

**Comment by DarkGamingDronze** [comment]
Says all checks have passed

Bump

**Comment by Wild1145** [comment]
@DarkGamingDronze This currently does not meet the Contributing guidelines, please see my earlier comment.

**Comment by DarkGamingDronze** [comment]
@Wild1145, I have fixed it now. Sorry for the long time.

**Comment by Wild1145** [comment]
@DarkGamingDronze It still requires a issue number, like #1234 for the issue that was created.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_wildcard.java [hunk 1]
```diff
@@ -38,7 +47,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         if (BLOCKED_COMMANDS.contains(runCmd.getName()))
         {
-            msg("Did you really think that was going to work?", ChatColor.RED);
+            msg("Did you really think that was going to work? What where you thinking?", ChatColor.RED);
```
Comment (Wild1145) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_wildcard.java:50]: Probably would change this message all together myself. More along the lines of "Sorry that command cannot be used with the Wildcard command. Please try something else"

**Review by Wild1145 — CHANGES_REQUESTED** [review]
Minor cosmetic change requested.

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_wildcard.java (hunk 1 above)
Comment (JeromSar) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_wildcard.java:50]: Originally just @StevenLawson's humor there. :)

**Review by Wild1145 — APPROVED** [review]

**Comment by Wild1145** [comment]
Thanks for contributing. I am closing this issue for inactivity. Should you have anything further to add in the future please either re-open the issue yourself or tag a member of staff and we will re-open the issue for further discussion.

**Pull request closed without merging**